### ISSUE TITLE
Getting Classes from Training Test

### ISSUE THREAD

**Issue by eanmikale** [dusty-nv/jetson-inference#1372]
Dusty, 
During training, it would be extremely useful if we can see which classes are underperforming during testing, so we may more efficiently adjust data-sets between training. Is there a way to do this with the jetson.utils library? Thank you.

**Comment by dusty-nv** [comment]
Hi @eanmikale, that wouldn't be a jetson.inference/jetson.utils feature, because those libraries are used for runtime inferencing only.   This would be something to be added to the PyTorch training scripts.  In theory it's possible, but I haven't looked into it.  I get the PyTorch training scripts from other sources.

**Comment by eanmikale** [comment]
For anyone else who might be interested, it seems this might be the closest we can gethttps://stackoverflow.com/questions/62958248/calculate-accuracy-for-each-class-using-cnn-and-pytorch. We can close it as far as I am concerned. Thanks Dusty.  ---- On Wed, 23 Feb 2022 12:48:36 -0600  ***@***.***  wrote ----
Hi @eanmikale, that wouldn't be a jetson.inference/jetson.utils feature, because those libraries are used for runtime inferencing only.   This would be something to be added to the PyTorch training scripts.  In theory it's possible, but I haven't looked into it.  I get the PyTorch training scripts from other sources.
